{
  "id": "CVE-2021-4118",
  "tags": [
    {
      "label": "Product",
      "message": "pytorchlightning/pytorch-lightning"
    },
    {
      "label": "Version",
      "message": "< 1.6 "
    },
    {
      "label": "Vulnerability",
      "message": "CWE-502 Deserialization of Untrusted Data"
    }
  ],
  "description": " pytorch-lightning is vulnerable to Deserialization of Untrusted Data",
  "poc": {
    "references": [
      "https://huntr.dev/bounties/31832f0c-e5bb-4552-a12c-542f81f111e6"
    ]
  }
}
